config/rubocop.yml

```diff
@@ -0,0 +1,56 @@
+require: rubocop-rspec
```

Comment:
We have a rubocop.yml file in the root of the project, shall we delete that in favour of this file?